controllers/app.controllers.js

```diff
@@ -30,8 +29,8 @@ const getEndPoints = (_, response, next) => {
 const getArticleById = (request, response, next) => {
   const { article_id } = request.params;
   fetchArticleById(article_id)
-    .then((data) => {
-      response.status(200).send({ article: data });
+    .then((article) => {
+      response.status(200).send({ article: article });
```

Comment:
this is a nice syntax change! 👌 You go a step further, as it's the same name for both the key and value you can just have 
`.send({ article });`